examples/allocative.rs

```diff
@@ -0,0 +1,40 @@
+//! Example for using [allocative::Allocative]
+
+#[cfg(not(feature = "allocative"))]
+fn main() {
+	panic!("re-run with --features allocative")
+}
+
+#[cfg(feature = "allocative")]
+fn main() {
+	use allocative::Allocative;
+	use allocative::FlameGraphBuilder;
+	use miny::Miny;
+
+	#[derive(Allocative)]
+	struct Example {
+		int: Miny<i32>,
+		trait_object_inline: Miny<dyn Allocative + Sync>,
+		trait_object_allocated: Miny<dyn Allocative + Sync>,
+	}
+
+	let example = Miny::new(Example {
+		int: Miny::new(0),
+		trait_object_inline: Miny::new_unsized(0i32),
+		trait_object_allocated: Miny::new_unsized(String::from("contents here")),
```

Comment:
`String` might be a bad example since it has its own pointer anyways, maybe a large array (e.g. `[0_i32; 100]`) could be more demonstrative